File: internal/radio/other_imf_data.go
package radio

type OtherImfData struct {
	TheRangeOfVHF    int    `json:"theRangeOfVHF"`
	TheMinFreqOfVHF  string `json:"theMinFreqOfVHF"`
	TheMaxFreqOfVHF  string `json:"theMaxFreqOfVHF"`
	TheRangeOfUHF    int    `json:"theRangeOfUHF"`
	TheMinFreqOfUHF  string `json:"theMinFreqOfUHF"`
	TheMaxFreqOfUHF  string `json:"theMaxFreqOfUHF"`
	EnableTxUHF      bool   `json:"enableTxUHF"`
	EnableTxVHF      bool   `json:"enableTxVHF"`
	PowerUpChar1     string `json:"powerUpChar1"`
	PowerUpChar2     string `json:"powerUpChar2"`
	EnableTxOver480M bool   `json:"enableTxOver480M"`
}

func CreateOIMFData() OtherImfData {
	return OtherImfData{
		TheRangeOfVHF:    0,
		TheMinFreqOfVHF:  "136",
		TheMaxFreqOfVHF:  "174",
		TheRangeOfUHF:    0,
		TheMinFreqOfUHF:  "400",
		TheMaxFreqOfUHF:  "520",
		EnableTxUHF:      true,
		EnableTxVHF:      true,
		PowerUpChar1:     "BAOFENG",
		PowerUpChar2:     "UV-5R",
		EnableTxOver480M: false,
	}
}

File: lib/otherimfdata.cs
namespace SQ5R
{
    [Serializable]
    public class OtherImfData
    {
        public int theRangeOfVHF = 0;

        public string theMinFreqOfVHF = "136";

        public string theMaxFreqOfVHF = "174";

        public int theRangeOfUHF = 0;

        public string theMinFreqOfUHF = "400";

        public string theMaxFreqOfUHF = "520";

        public bool enableTxUHF = true;

        public bool enableTxVHF = true;

        public string powerUpChar1 = "BAOFENG";

        public string powerUpChar2 = "UV-5R";

        public bool enableTxOver480M = false;
    }
}